core/templates/dev/head/pages/error/error.html

```diff
@@ -67,6 +67,9 @@ <h2 class="oppia-error-h2" translate="I18N_ERROR_MESSAGE_500"></h2>
     .oppia-footer {
       position: absolute;
     }
+    .oppia-wide-panel-content {
+      margin-bottom: 240px;
```

Comment:
The desktop view seems fine now but the mobile view carries too much bottom margin.

See below: 
![screenshot from 2017-05-25 16-25-12](https://cloud.githubusercontent.com/assets/24438869/26447655/16568360-4167-11e7-804b-67745408d61b.png)